src/rest-server/src/utils/error.d.ts

```diff
@@ -42,6 +42,7 @@ declare type Code =
     'NoGroupError' |
     'NoStorageError' |
     'NoVirtualClusterError' |
+    'NoRequestError' |
```

Comment:
the name is strange, sounds like no http request received

Reply:
Change to `NoUserRequestError`.